co-log-core/src/Colog/Core/Action.hs

```diff
@@ -575,3 +579,28 @@ multiplicate (LogAction l) = LogAction $ \msgs -> l (fold msgs)
 {-# INLINE multiplicate #-}
 {-# SPECIALIZE multiplicate :: Monoid msg => LogAction m msg -> LogAction m [msg] #-}
 {-# SPECIALIZE multiplicate :: Monoid msg => LogAction m msg -> LogAction m (NonEmpty msg) #-}
+
+{- | Allows changing the internal monadic action.
+
+Let's say we have a pure logger action using 'PureLogger'
+and we want to log all messages into 'IO' instead.
+
+If we provide the following function:
+
+@
+performPureLogsInIO :: PureLogger a -> IO a
+@
+
+then we can convert a logger action that uses a pure monad
+to a one that performs the logging in the 'IO' monad using:
+
+@
+hoistLogAction performPureLogsInIO :: LogAction (PureLogger a) a -> LogAction IO a
```

Comment:
Minor documentation typo :slightly_smiling_face: 
```suggestion
hoistLogAction performPureLogsInIO :: LogAction PureLogger a -> LogAction IO a
```

Reply:
Sorry, just to better understand your point:

I thought that `PureLogger` in the `Colog.Pure` module always has a message argument to go along with it, for example here: https://github.com/kowainik/co-log/blob/e848fed2939a29c4293f6fcf5de6f547882e4d4a/co-log/test/Property.hs#L27

So that's why I'd assume that the type of a pure logger action should be `LogAction (PureLogger msg) msg`.

Could you please explain why is my understanding incorrect? :)

Reply:
@jiribenes Your understanding is 100% correct :ok_hand: I just had too much PRs to review and missed some parts of the implementation. There's no type-checker for the documentation, so it won't harm to be extra careful.